content/3_Silver/Sorting_Custom.mdx

```diff
@@ -1176,3 +1178,50 @@ Many of the problems below may use other Silver concepts, such as
 </Info>
 
 <Problems problems="general" hideSuggestProblemButton />
+
+## Quiz
+
+<Quiz>
+  <Quiz.Question>
+    When are custom comparators used?
```

Comment:
ok i don't really see the point of this quiz

cuz like, a custom comparator is something where you just go like "oh I need to sort these things in a non-predefined way time to write a comparator", and isn't any more complicated than that

Reply:
I guess it might be too obvious. I'll alter the question.